test/system_test.py

```diff
@@ -193,6 +204,13 @@ def test_install_libfyaml_default():
     spack_install('libfyaml')
 
 
+@pytest.mark.no_tsa  # No uenv on Tsa
+@pytest.mark.no_daint  # No uenv on Daint
+@pytest.mark.libfyaml
+def test_install_libfyaml_default_uenv(prepost):
+    spack_install('libfyaml', uenv=prepost)
```

Comment:
```
@pytest.mark.no_tsa  # No uenv on Tsa
@pytest.mark.no_daint  # No uenv on Daint
@pytest.mark.libfyaml
def test_install_libfyaml_default_uenv(prepost):
    spack_install('libfyaml', uenv=prepost)
```
has a very low signal-to-noise ratio.

Let's have a look.
Signal (= stuff that's actually useful to a developer):
```
                       no_tsa  # No uenv on Tsa
                       no_daint  # No uenv on Daint


    spack_install('libfyaml', uenv=prepost)
```

Noise (= stuff that's not important for a developers. It's only there for the machine):
```
@pytest.mark.
@pytest.mark.
@pytest.mark.libfyaml
def test_install_libfyaml_default_uenv(prepost):

```

Reply:
IMHO this is so noisy we need to find a better solution.

Reply:
Postpone a decision for now. Revisit once more tests actually needed for uenvs.